config/checkstyle/checkstyle.xml

```diff
@@ -184,7 +184,7 @@
             <property name="braceAdjustment" value="0"/>
             <property name="caseIndent" value="0"/>
             <property name="throwsIndent" value="4"/>
-            <property name="lineWrappingIndentation" value="4"/>
+            <property name="lineWrappingIndentation" value="8"/>
```

Comment:
![dg](https://user-images.githubusercontent.com/60384540/97528513-e5984200-19e8-11eb-93ce-3feb277c339d.jpg)


The above top class design is too messy with arrow and it might confused the reader. A  proper arrow head would improve readibility to the readers about your application.

The circle portion of the diagram I have marked may be hard for user to get a better picture of the flow of your design.
Perharps, use straight lines or adjust the different box accordingly so that user can follow through the flow of the design